site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -6,8 +6,9 @@ description: >
   Learn how to configure your Agones services for high availability and resiliancy to disruptions.
 publishDate: 2023-02-28
 ---
-
-{{< alpha title="split controller and extensions" gate="SplitControllerAndExtensions" >}}
+{{% feature publishVersion="1.32.0" %}}
+{{< beta title="split controller and extensions" gate="SplitControllerAndExtensions" >}}
+{{% /feature %}}
```

Comment:
I think you need a `feature expiryVersion=` block here as well, i.e. alpha expires at 1.32 and beta is published at 1.32.

Reply:
Done!